dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -26,21 +28,66 @@ new DotcomRendering(app, 'DotcomRendering-CODE', {
 	instanceType: 't4g.micro',
 });
 
-new DotcomRendering(app, 'DotcomRendering-front-web-CODE', {
-	...sharedProps,
-	app: 'front-web',
+/** NEW article stack */
+new RenderingCDKStack(cdkApp, 'ArticleRendering-CODE', {
+	guApp: 'article-rendering',
 	stage: 'CODE',
-	minCapacity: 1,
-	maxCapacity: 4,
-	instanceType: 't4g.micro',
+	domainName: 'article-rendering.code.dev-gutools.co.uk',
+	scaling: { minimumInstances: 1, maximumInstances: 2 },
+	instanceSize: InstanceSize.MICRO,
 });
-
-new DotcomRendering(app, 'DotcomRendering-front-web-PROD', {
-	...sharedProps,
-	app: 'front-web',
+new RenderingCDKStack(cdkApp, 'ArticleRendering-PROD', {
+	guApp: 'article-rendering',
 	stage: 'PROD',
-	// TODO: up this once we have code working
-	minCapacity: 1,
-	maxCapacity: 4,
-	instanceType: 't4g.micro',
+	domainName: 'article-rendering.gutools.co.uk',
```

Comment:
Not a blocker: Not sure why we are choosing gutools here (same question for CODE app)?

Reply:
I don't have a preference tbh! Just thought that usually `gutools` is used in our tools services

Reply:
I think Ioanna is right about that convention, I think in this case we'd use .guardianapis.com - if it's not too much hassle to change

Reply:
I'm not entirely sure about this either, maybe devX would be more familiar with the conventions

Reply:
More pedantry but I think `guardianapis` is intended for backend services, although this distinction is less clear in the context of DCR being a service for Frontend.

There's also `guardianapps.co.uk` which might be a better fit but devX will know definitively.